strawberry/types/info.py

```diff
@@ -15,7 +17,9 @@ class Info(Generic[ContextType, RootValueType]):
     context: ContextType
     root_value: RootValueType
     variable_values: Dict[str, Any]
+    return_type: Optional[Union[Type, StrawberryUnion]]
     # TODO: create an abstraction on these fields
     operation: OperationDefinitionNode
     path: Path
-    # TODO: parent_type, return_type as strawberry types
+    # TODO: parent_type as strawberry types
+    original_info: GraphQLResolveInfo
```

Comment:
do we need this? 🤔

Reply:
It depends, I guess until we expose all the information having the GraphQL info might be helpful?

Reply:
I'd just remove it for now and implement the remaining bits in another pr :)